ecommerce/frontend/src/components/TrendingGrid.js

```diff
@@ -1,23 +1,24 @@
 import './TrendingGrid.less'
+import { Link } from 'react-router-dom'
+
+export const TrendingGrid = trendingProducts => {
+    var gallerytItemNumber = 0
+    function gallerytItemNo() {
+        gallerytItemNumber += 1
+        return gallerytItemNumber
+    }
+    const products = trendingProducts.trendingProducts
+    console.log(products)
```

Comment:
please call the parameter `props` instead of `trendingProducts` as that is the convention.

Or you can directly extract the trending products as :
```js
export const TrendingGrid = ({ trendingProducts}) => {
...
}
```

Also please remove the console.log